app/controllers/api/v1/admin/users_controller.rb

```diff
@@ -4,6 +4,26 @@ module Api
   module V1
     module Admin
       class UsersController < ApiController
+        include Avatarable
+
+        def active_index
+          # TODO: Change to get active users only
+          users = User.all.to_a
+
+          users.map! do |user|
+            {
+              id: user.id,
+              name: user.name,
+              email: user.email,
+              provider: user.provider,
+              role: Role.find(user.role_id).name,
```

Comment:
You can use user.role.name

Reply:
Could we call this active_users instead of active_index? And then pending_users, banned_users, deleted_users